src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_CertificateReplacement.swift

```diff
@@ -0,0 +1,51 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+
+class ENAUITests_CertificateReplacement: CWATestCase {
```

Comment:
Maybe we should use the same naming here as well and the test plan could also be called "HealthCertificateReiisuance" or "ReissueHealthCertificate"
```suggestion
class ENAUITests_CertificateReissuance: CWATestCase {
```